react/ConditionLayoutBinding.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useMemo } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import ConditionLayout from './ConditionLayout'
+import type { NoUndefinedField, MatchType, Condition, Handlers } from './types'
+
+type Props = {
+  conditions: Array<Condition<ContextValues, HandlerArguments>>
+  matchType?: MatchType
+  Else?: React.ComponentType
+  Then?: React.ComponentType
```

Comment:
What do you think about this changes?
```tsx
// add on top of this file
import type { ComponentType } from 'react'
```

```suggestion
  Else?: ComponentType
  Then?: ComponentType
```